test/server/test_file_management.py

```diff
@@ -142,6 +142,7 @@ def restart_server(**extra_env):
     env['DEBUG_GOOGLE_ANALYTICS'] = ''
     env['NODE_ENV'] = 'production'
     env['LOG_QUERY_LIMIT'] = '0'
+    env['ENABLE_WATCHDOG'] = 'false'
```

Comment:
Need to set this because the default is true but the server would halt because other config values do not exist.